sjb/config/test_cases/test_pull_request_origin_service_catalog.yml

```diff
@@ -0,0 +1,115 @@
+---
+parent: 'common/test_cases/origin.yml'
+overrides:
+  junit_analysis: False
+extensions:
+  sync_repos:
+    - name: "openshift-ansible"
```

Comment:
can remove this line

Reply:
I removed only the line containing  - name: "openshift-ansible"